Product/Production/Adapters/General/CONNECTAdminGUI/src/main/webapp/certificateManager.xhtml

```diff
@@ -348,7 +348,7 @@
                                             <p:autoUpdate />
                                         </p:messages>
                                         <div class="content">
-                                            <h:outputText>Creating a certificate creates a temporary certificate. This temporary certificate will be used to replace with the existing keystore file. Please skip to Create CSR if existing certificate will be used.</h:outputText>
+                                            <h:outputText>This creates a temporary certificate that will be used to replace existing keystore file. Please skip to Create CSR if existing certificate will be used.</h:outputText>
```

Comment:
"Please skip to Create CSR if existing certificate will be used." ->"Please skip to Create CSR if an existing certificate will be used."